### PR TITLE
fix: APPS-2827 Component SectionTeaserCard

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-2827](https://jira.library.ucla.edu/browse/APPS-2827)

1. Deletes the Section Title in the `SectionTeaserCard` because we are adding it on the page instead.
2. Adds rounded edges to the card
3. It deletes the calendar icon so there can be more space around the date
4. Adds 3 lines max truncation to the title
5. Moves the Date & Time to the bottom of the card
6. Makes all cards same height  min-height 350px
    Delete this style on the page
7. Change the hovering border to a box-shadow
8. Update stories
9. Changes the scrollbar color

- https://deploy-preview-577--ucla-library-storybook.netlify.app/?path=/story/block-card-with-image--ftva-items&globals=backgrounds.value:!hex(0B6AB7)
- https://deploy-preview-577--ucla-library-storybook.netlify.app/?path=/story/block-card-with-image--ftva-items-no-image&globals=backgrounds.value:!hex(0B6AB7)
- https://deploy-preview-577--ucla-library-storybook.netlify.app/?path=/story/section-teaser-card--ftva-with-section-title&globals=backgrounds.value:!hex(0B6AB7)

Screenshot 2024-08-02 at 6 09 44 PM

[APPS-2827]: https://uclalibrary.atlassian.net/browse/APPS-2827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/stories/SectionTeaserCard.stories.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const mock = [
     image: API.image,
     to: '/visit/foo/barf/',
     category: 'Curabitur ',
-    title: 'Vel Quam Elementum',
```
Comment (farosFreed) [inline src/stories/SectionTeaserCard.stories.js:55]: Why the change?

File: src/lib-components/BlockCardWithImage.vue [hunk 2]
```diff
@@ -130,10 +140,13 @@ const parsedDateFormat = computed(() => {
         <MoleculePlaceholder
           class="molecule"
           aria-hidden="true"
+          :aspect-ratio="imageAspectRatio"
```
Comment (farosFreed) [inline src/lib-components/BlockCardWithImage.vue:143]: does this have any effect? I don't think this SVG has any implementation for aspect-ratio?

File: src/stories/SectionTeaserCard.stories.js [hunk 3]
```diff
@@ -104,16 +150,16 @@ export function Default() {
   }
 }
 
-const FTVAMock = mock.slice(3, 6).map((item) => {
-  return {
-    ...item,
-    category: ''
-  }
-})
+// const FTVAMock = mock.slice(3, 6).map((item) => {
```
Comment (farosFreed) [inline src/stories/SectionTeaserCard.stories.js:153]: we can delete this if you don't need it

File: src/stories/SectionTeaserCard.stories.js (hunk 1 above)
Comment (farosFreed) [inline src/stories/SectionTeaserCard.stories.js:55]: I am curious why we need this change? Do longer strings break new styles?

File: src/stories/SectionTeaserCard.stories.js (hunk 3 above)
Comment (farosFreed) [inline src/stories/SectionTeaserCard.stories.js:153]: can remove this if we're not using it

File: src/styles/ftva/_block-card-with-image.scss [hunk 4]
```diff
@@ -1,56 +1,107 @@
...
+        bottom: 0;
+    }
+
+    .only-date-time {
+        display: flex;
+        flex-direction: row;
+
+        margin-bottom: 10px;
+
+        //background-color: aquamarine;
```
Comment (farosFreed) [inline src/styles/ftva/_block-card-with-image.scss:46]: was this left from before or new? Can we clear it out?

File: src/styles/ftva/_block-card-with-image.scss [hunk 5]
```diff
@@ -1,56 +1,107 @@
...
     }
+
+    // Hovers
+    @media #{$has-hover} {
+        &:hover {
+            box-shadow: 0 4px 4px rgb(51 51 51 / 0.2);
+        }
+    }
+
+    .ftva.section-teaser-card .card[data-v-a9e65b06]:hover {
```
Comment (farosFreed) [inline src/styles/ftva/_block-card-with-image.scss:98]: are these hover styles being applied? I don't see blue borders on hover, only the box shadow

**Review by tinuola — DISMISSED** [review]

File: src/styles/ftva/_block-card-with-image.scss [hunk 6]
```diff
@@ -1,56 +1,107 @@
...
+        /* border: 2px solid #115daf; */
+        box-shadow: 0 10px 17px rgba(0, 0, 0, 0.1);
+    }
+
+    .ftva.section-teaser-card .card[data-v-49e2255f]:hover {
+        border: 2px solid #115daf;
```
Comment (farosFreed) [inline src/styles/ftva/_block-card-with-image.scss:104]: same with this. Can this rule be removed? Its odd to see the [data-v-49e2255f] in the styles, I believe these numbers change each time we export the code so these styles wont stick

File: src/lib-components/BlockCardWithImage.vue [hunk 7]
```diff
@@ -152,6 +165,9 @@ const parsedDateFormat = computed(() => {
   </li>
 </template>
 
-<style lang="scss" scoped>
+<style
+  lang="scss"
+  scoped
+>
 @import "@/styles/themes.scss";
```
Comment (farosFreed) [inline src/lib-components/BlockCardWithImage.vue:172]: it might be worth refactoring this import to just import the component specific files like sectionteasercard.vue is doing. This will prevent bloat in the components and might help with some style clashing in cardmeta and blockcardwithimage

**Review by farosFreed — CHANGES_REQUESTED** [review]
Looks good, I left a few comments about bits that looked like they might not be doing anything to me. Can we clean these up?

File: src/stories/SectionTeaserCard.stories.js (hunk 1 above)
Comment (jendiamond) [reply src/stories/SectionTeaserCard.stories.js:55]: I was just checking that this was working.

File: src/lib-components/BlockCardWithImage.vue (hunk 2 above)
Comment (jendiamond) [reply src/lib-components/BlockCardWithImage.vue:143]: yeah, I'll take that out. I was having an issue with it expanding vertically so I tried this to see if it worked. I ended up handling it in the CSS. I forgot to take this out.

File: src/styles/ftva/_block-card-with-image.scss (hunk 5 above)
Comment (jendiamond) [reply src/styles/ftva/_block-card-with-image.scss:98]: Yes, these are extra.

File: src/styles/ftva/_block-card-with-image.scss (hunk 6 above)
Comment (jendiamond) [reply src/styles/ftva/_block-card-with-image.scss:104]: Yes this is unnecessary

File: src/lib-components/BlockCardWithImage.vue (hunk 7 above)
Comment (jendiamond) [reply src/lib-components/BlockCardWithImage.vue:171]: We decided to make a bug ticket for this because the change affects all the styles.

**Review by farosFreed — APPROVED** [review]
